src/components/molecules/Darkmode/DarkmodeContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import storage from '../../../lib/storage'
+import Darkmode from './Darkmode'
+import { compose, withStateHandlers, withPropsOnChange, withHandlers, lifecycle, pure} from 'recompose';
+
+export default compose(
+  pure,
+  withStateHandlers(
+    {
+      isActive: false,
+      isSupported: null
+    },
+    {
+      toggle: ({isActive}) => (value) => ({
+        isActive: value
+      })
+    }
+  ),
+  withHandlers({
+    isSupported: () => (property = 'filter', value = 'invert(100%)') => {
+      let prop = property + ':',
+      el = document.createElement('test'),
+      mStyle = el.style
+      el.style.cssText = prop + value
+      return mStyle[property]
+    }
```

Comment:
@Joe-Pena We will need to change the name of this handler if we have state called `isSupported`. There is a potential to over ride the state with the handler and cause bugs.